Libraries/Utilities/MessageQueue.js

```diff
@@ -182,18 +188,22 @@ class MessageQueue {
       this._lastFlush = now;
     }
     Systrace.counterEvent('pending_js_to_native_queue', this._queue[0].length);
-    if (__DEV__ && SPY_MODE && isFinite(module)) {
-      console.log('JS->N : ' + this._remoteModuleTable[module] + '.' +
```

Comment:
Maybe MessageQueue.spy(true) to turn on the default function, MessageQueue.spy(fn) to provide a custom function and MessageQueue.spy(false) to turn it off?